app/connector/BindingTariffClassificationConnector.scala

```diff
@@ -73,34 +78,43 @@ class BindingTariffClassificationConnector @Inject() (
     s"${appConfig.bindingTariffClassificationUrl}/cases?$queryString"
   }
 
-
   def findCasesByQueue(
     queue: Queue,
     pagination: Pagination,
-    types: Seq[ApplicationType] = Seq(ApplicationType.ATAR, ApplicationType.LIABILITY, ApplicationType.CORRESPONDENCE)
+    types: Seq[ApplicationType] = Seq(
+      ApplicationType.ATAR,
+      ApplicationType.LIABILITY,
+      ApplicationType.CORRESPONDENCE,
```

Comment:
Would it be cleaner to use ApplicationType.values.toSeq here instead of a Seq listing all individual types?

Reply:
Indeed, that is a good suggestion, thanks 👍